### PR TITLE
Exposing supported methods used by InteropService

### PR DESCRIPTION
**Pull request by lock9**
I'm using this in neo-cli to see the syscalls available.

### CODE REVIEW

**Comment by erikzhang** [comment]
You can use reflection to get these methods.

**Comment by lock9** [comment]
But shouldn't the InteropService expose the method it has implemented?
I would like to convert the hashes into names. I'm using it to debug neo3 with some additional changes I've made in neo-cli:
image

**Comment by shargon** [comment]
> I would like to convert the hashes into names

Is not possible to revert a hash, you need to cache the name first

**Comment by lock9** [comment]
@shargon that is why we need this PR 😂

File: neo/SmartContract/InteropService.cs [hunk 1]
```diff
@@ -77,6 +77,16 @@ public static long GetPrice(uint hash, RandomAccessStack<StackItem> stack)
             return methods[hash].GetPrice(stack);
         }
 
+        public static Dictionary<uint, string> SupportedMethods()
```
Comment (shargon) [inline neo/SmartContract/InteropService.cs]: Is the same dictonary?

**Pull request merged**